tests/sles4sap/qesapdeployment/test_cluster.pm

```diff
@@ -19,7 +18,7 @@ sub run {
 
     qesap_ansible_cmd(cmd => $_, provider => $prov, $_) for ('pwd', 'uname -a', 'cat /etc/os-release');
     qesap_ansible_cmd(cmd => 'ls -lai /hana/', provider => $prov, filter => 'hana');
-    qesap_ansible_cmd(cmd => $_, provider => $prov, filter => 'vmhana01'), for ('crm status', 'crm_mon -R -r -n -N -1');
+    qesap_ansible_cmd(cmd => $_, provider => $prov, filter => 'vmhana01'), for ('crm status', $crm_mon_cmd);
```

Comment:
Nice! Thank you!

Reply:
This code is fine thanks to `use hacluster '$crm_mon_cmd';`

Using a more generic `use hacluster;` result in an error about a double definition of post_fail_hook

Reply:
Makes sense. You could've probably also used `$hacluster::crm_mon_cmd`.